index.js

```diff
@@ -63,6 +71,15 @@ function createFromElement(el) {
 function getElementProperties(el) {
   var obj = {}
 
+  // IE8 data attributes
+  if (dataAttrs.length > 0) {
+    dataAttrs.forEach(function(dataAttr) {
+      if (el.getAttribute(dataAttr)) {
+        obj[dataAttr] = el.getAttribute(dataAttr);
```

Comment:
It seems to work, here is the test case: https://github.com/agentcooper/vdom-virtualize/blob/master/specs/spec/spec.js#L55